integration-cli/docker_cli_save_load_test.go

```diff
@@ -167,6 +167,20 @@ func (s *DockerSuite) TestSaveAndLoadRepoFlags(c *check.C) {
 	c.Assert(before, checker.Equals, after, check.Commentf("inspect is not the same after a save / load"))
 }
 
+func (s *DockerSuite) TestSaveWithNoExistImage(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+	name := "test-save-image"
+	dockerCmd(c, "run", "--name", name, "busybox", "true")
```

Comment:
just test we did have an image, and delete it, so that image not exist, and then run save would get the expected error  :)